config/sidekiq.yml

```diff
@@ -1,4 +1,4 @@
-:concurrency: 5
+:concurrency: 10
```

Comment:
I don't get why we want to increase concurrency, can you explain?

Reply:
Probably not the good way to test it, but exploring Sidekiq configurations I found 10 threads is the default conf. 

So I wonder why do we need to reduce to 5 rather than keep the default to 10

Reply:
Yes but in this case the connection pool needs to be increased as well.

https://dev.to/appaloosastore/active-record-sidekiq-pools-and-threads-18d5

As this is not an issue and workds perfecxtly on many instances, I suggest we stick to 5 threads.

However, i suggest to reorganize queues to improve performances on critical queues.